components/threading/global_threads/global_threads.tsx

```diff
@@ -0,0 +1,3 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
```

Comment:
Why this empty file?

Reply:
Just forgot to delete after init scaffolding, deleted now